src/AltinnCore/Common/Services/Implementation/AuthorizationAppSI.cs

```diff
@@ -3,7 +3,10 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
 using System.Threading.Tasks;
+using Altinn.Authorization.ABAC.Xacml.JsonProfile;
```

Comment:
Same here, should we rather implement all these changes in the template-project as it will have to be moved into that solution shortly anyways?